doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+
+The current release of GTDB is [**R04-RS89**](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/) (version 4, indexed to RefSeq release 89), when this tutorial is written, and on which the tutorial is based. This release contains 143,512 bacterial and 2,392 archaeal organisms in its taxonomy, or 23,458 bacterial and 1,248 archaeal genomes in its collection of actual genome sequences. These 24,706 "**species clusters**" cover the complete GTDB taxonomic framework ([Parks et al., 2020](https://www.nature.com/articles/s41587-020-0501-8)).
+
+This tutorial will explain various strategies of utilizing GTDB taxonomy / phylogeny in a meta'omics analysis with Woltka.
+
+
+## Contents
+
+- [Woltka directly parses GTDB taxonomy](#woltka-directly-parses-gtdb-taxonomy)
+- [Reformat GTDB as taxdump style](#reformat-gtdb-as-taxdump-style)
+- [Build GTDB genome database](#build-gtdb-genome-database)
+- [Use GTDB phylogeny](#use-gtdb-phylogeny)
+- [Use WoL database with GTDB taxonomy](#use-wol-database-with-gtdb-taxonomy)
+
+
+## Woltka directly parses GTDB taxonomy
+
+The GTDB taxonomy ([`bac120_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/bac120_taxonomy_r89.tsv) and [`ar122_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/ar122_taxonomy_r89.tsv)) is provided in the format of **Greengenes-style lineage strings**. For example: _Escherichia coli_ O157:H7 str. Sakai (NCBI RefSeq assembly: [GCF_000008865.1](https://www.ncbi.nlm.nih.gov/assembly/GCF_000008865.1/)) is classified as:
+
+```
+RS_GCF_000008865.1	d__Bacteria;p__Proteobacteria;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Escherichia;s__Escherichia dysenteriae
+```
+
+This format can be directly parsed by Woltka:
+
+```bash
+woltka classify --lineage bac120_taxonomy_r89.tsv ...
+```
+
+If you work with both Bacteria and Archaea (this is the common case), you may simply concatenate the two taxonomy files before use:
+
+```bash
+cat bac120_taxonomy_r89.tsv ar122_taxonomy_r89.tsv > gtdb_taxonomy.tsv
+```
+
+Note: The [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk) database ([`gtdbtk_data.tar.gz`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/gtdbtk_r89_data.tar.gz)) already provides a combined `gtdb_taxonomy.tsv` file.
+
+
+## Reformat GTDB as taxdump style
+
+Alternatively, you may consider converting GTDB taxonomy into the format of NCBI taxdump. This will allow you to connect GTDB to more bioinformatics tools such as BLAST and Kraken2 which favor the taxdump format. This will also make the Woltka output slimer, if that's your preference. For example, instead of the long lineage strings like `d__Bacteria;p__Proteobacteria;...`, you will get taxon names like `Escherichia` as the feature name.
```

Comment:
> Alternatively, you may consider converting GTDB taxonomy into the format of NCBI taxdump.

Perhaps can be written: 

> Alternatively, you may consider converting a GTDB taxonomy format into a NCBI taxdump format.

Reply:
slimer should be slimmer

Reply:
Hmmm... "GTDB taxonomy" is a specific thing. There is only one "GTDB taxonomy". Maybe adding "the" will work better? Also I am not sure if "format" can be singular or plural (but I could be wrong...). Changed slimer into slimmer.